sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs

```diff
@@ -144,11 +143,23 @@ private AccessToken GetSasTokenAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
+        /// Gets an Azure.Core.AccessToken for the specified set of scopes.
+        /// </summary>
+        /// <param name="requestContext">The Azure.Core.TokenRequestContext with authentication information.</param>
+        /// <param name="cancellationToken">The System.Threading.CancellationToken to use.</param>
+        /// <returns>     A valid Azure.Core.AccessToken.</returns>
```

Comment:
Why extra space after the returns start tag?

Reply:
There is no need. Will fix the spacing.